designer-demo/env/.env.development

```
@@ -5,4 +5,4 @@ VITE_CDN_DOMAIN=https://registry.npmmirror.com
 # 使用npmmirror的cdn 时，需要声明 VITE_CDN_TYPE=npmmirror
 VITE_CDN_TYPE=npmmirror
 # request data via alpha service
-# VITE_ORIGIN=
+# VITE_ORIGIN=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)